### PR TITLE
replace github.com/negbie with github.com/sipcapture for all package …

### PR DESCRIPTION
**Pull request by seanchann**
The currently known problem is that no `package logp` was found. where can I to check out it

### CODE REVIEW

**Comment by negbie** [comment]
@seanchann I guess this will brake more. For example 
dotsql and logp are helper packages which are under
github.com/negbie/dotsql
github.com/negbie/logp

Does following not work for you?
go get -u -v -d github.com/sipcapture/heplify-server
cd go/src/github/sipcapture/heplify-server/cmd/heplify-server
go get -u -v -d
go build

**Comment by seanchann** [comment]
modify `logp` and `dotsql` package import error.

File: build/image/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,34 @@
+FROM golang:alpine as builder
+
+LABEL maintainer="zhouxq@ti-net.com.cn"
+
+RUN set -ex \
+  && apk add --update  bash\
+  && rm -rf /var/cache/apk/*
```
Comment (mauri870) [inline build/image/Dockerfile]: Since this is a builder stage, you don't need to clean the cache at all.

File: build/image/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+ENV BUILD_ROOT=/go/src/github.com/sipcapture/heplify-server/
```
Comment (mauri870) [inline build/image/Dockerfile]: Now after Go 1.11+ it may throw an error if trying to use mdules inside $GOPATH

File: build/image/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+COPY . ${BUILD_ROOT} 
+
+RUN cd ${BUILD_ROOT} \
+  && cp vendor/* /go/src/ -rf \
+  && cd ${BUILD_ROOT}/cmd/heplify-server \
+  && CGO_ENABLED=0 GOOS=linux go build -v -a -ldflags '-s -w' -installsuffix cgo -o  /go/bin/heplify-server \
+  && rm -rf /go/src/
```
Comment (mauri870) [inline build/image/Dockerfile]: Any cleanup in a builder stage is not necessary since it'll be discarded.

File: deploy/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# 部署说明
```
Comment (mauri870) [inline deploy/README.md]: Roses are red, violets are blue, the repo is english so why aren't you? :notes:

Comment (seanchann) [reply deploy/README.md]: sorry , this part not need to merge here. I will close it

**Comment by negbie** [comment]
@seanchann You could create a branch with your changes and create a pull request. For example only with the changed imports.

**Comment by mauri870** [comment]
Yeah, it's better to split your pull request otherwise it'll be large and harder to review

**Pull request closed without merging**